code/game/turfs/simulated/water.dm

```diff
@@ -96,7 +96,7 @@
 
 /mob/living/carbon/human/can_breathe_water()
 	if(species)
-		return species.can_breathe_water()
+		return species.waterbreathing
```

Comment:
Please don't remove this abstraction. It is a step away from object oriented programming. Have the proc return this var if you want, and to be honest, give that to Polaris because it's an improvement in how it's done.

Reply:
So like.. 
``return species.can_breath_water(waterbreathing)`` ?

Reply:
No, like:
```
/mob/living/carbon/human/can_breathe_water()
    return species.can_breathe_water()
```
```
/datum/species/proc/can_breathe_water()
    return waterbreathing
```

That way if someone wants, they can do, like:
```
/datum/species/snowflake/can_breathe_water()
    if(some weird conditions)
        return TRUE
    else
        return FALSE
```